components/footer/footer.module.scss

```diff
@@ -1,10 +1,8 @@
 .infoRepo {
+  display: flex;
   font-weight: 100;
-  margin: 0;
   text-align: center;
-  padding: 15px 0px;
-  width: 100%;
-  height: 2.5rem;
-  position: absolute;
-  bottom: 0;
+  padding:15px 0px 5px;
+  flex-shrink: 0;
```

Comment:
What is the advantage of flex-shrink here?

Reply:
There is not any utility of this code here, I added it because it was a part of the code snippet which I followed. I will make changes